.github/workflows/docker-publish.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - 2u/project-theseus
```

Comment:
Can we remove docker-publish.yml for edx-internal? I suspect we may not want to publish Docker images for 2u/project-theseus.

Reply:
removed